requirements/requirements.txt

```diff
@@ -9,7 +9,7 @@ scikit-learn
 setuptools>=65.5.1
 tensorboard
 text-unidecode
-torch
+torch>=2.1.0
```

Comment:
Don't use > or any operator. It overrides container version. Revert

Reply:
It's cherry picked from #10079 to unblock me (CI is broken otherwise and I need to check these checks actually work). I will revert this change later but FYI it looks like this will be merged in another PR

Reply:
I think they reverted it. I don't have those changes anymore btw.

Reply:
Ah ok that's good